### ISSUE TITLE
Technical doubt regarding Logistic Regression

### ISSUE THREAD

**Issue by mayank2794** [aihubprojects/Logistic-Regression-From-Scratch-Python#2]
I am Mayank Agrawal from India, studying Machine Learning from your book titled "Essential Math for Data Science" by Thomas Nield. I was studying Logistic Regression topic and I am facing technical problem when I am writing code for the same in python from scratch.

Example referred from book is patient data given in Example 6-8.
Python-Code is attached herewith in .txt form
[patient_data.xlsx](https://github.com/aihubprojects/Logistic-Regression-From-Scratch-Python/files/12014822/patient_data.xlsx)
[Log-Reg.txt](https://github.com/aihubprojects/Logistic-Regression-From-Scratch-Python/files/12014823/Log-Reg.txt)
 for reference along with Input data excel file. I have used the matrix form of Gradient descent. but my answer is coming wrong as I am decreasing the learning rate.
The learning rate used in your example is 0.01, but my answer is coming correct only when learning rate is 0.1, else not. If I am decresing the learning rate to .01 answer is coming wrong.
Can you please help me regarding this.

Regards,
Mayank agrawal
India
